backend/services/operations_test.go

```diff
@@ -213,16 +212,16 @@ func TestUpdateOperation(t *testing.T) {
 		input := services.UpdateOperationInput{
 			OperationSlug: masterOp.Slug,
 			Name:          "New Name",
-			Status:        models.OperationStatusComplete,
 		}
-		require.NotEqual(t, masterOp.Status, input.Status)
+		// TODO TN - look into this?
+		// require.NotEqual(t, masterOp.Status, input.Status)
```

Comment:
It looks like you can safely remove this, and the similar status checks here

Reply:
oh whoops - forgot to remove those

Reply:
removed